conda-forge.yml

```diff
@@ -13,4 +13,7 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+remote_ci_setup:
+- conda-forge-ci-setup=4
+- conda-build=25.3
```

Comment:
Appears to be a conda-build 25.4 bug. Have filed upstream: https://github.com/conda/conda-build/issues/5704

For now rolled back to conda-build 25.3, which doesn't have this issue

We can unpin once there is a bug fix

Reply:
A fix has been added to `conda-build` (as noted in the upstream issue). However it is not yet released

To track reverting this once the `conda-build` fix is available, filed issue: https://github.com/conda-forge/certifi-feedstock/issues/110